certified-connectors/AIForged/Readme.md

```diff
@@ -0,0 +1,22 @@
+This is the AIForged connector based on the [AIForged REST API](https://portal.aiforged.com/swagger/index.html). It is based on OAUTH.  The connector allows automation and integration with AIForged
```

Comment:
@aiforged Please go thru the template given for the readme.md file and update it with the required sections.